docs/content/bart-tutorial.md

```diff
@@ -0,0 +1,150 @@
+title = "Building a site with Bartholomew"
```

Comment:
What is this file?
It seems to be the Spin URL shortener tutorial, but with a new title?

Reply:
As previously stated, this is the initial structure for the Bartholomew docs. This isn't the final product. I believe we should have a larger discussion about how we want things structured.